### PR TITLE
3.2 backports

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: test/lib/tasks/upgrades/3.0/update_subscription_facet_backend_data_test.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'katello_test_helper'
```
Comment (johnpmitsch) [inline test/lib/tasks/upgrades/3.0/update_subscription_facet_backend_data_test.rb]: @houndci-bot I disagree :angry:

**Comment by jlsherrill** [comment]
[test]

**Comment by jlsherrill** [comment]
Manual CI RUn: http://ci.theforeman.org/job/test_katello_pull_request/10135/

**Comment by ehelms** [comment]
[test]

**Comment by johnpmitsch** [comment]
Merged as the manual CI run was green

**Pull request merged**